src/core/src/core_logic/RebootManager.py

```diff
@@ -91,6 +91,7 @@ def start_reboot_if_required_and_time_available(self, current_time_available):
         """ Starts a reboot if required. Happens only at the end of the run if required. """
         self.composite_logger.log("\nReboot Management")
         reboot_pending = False if not self.status_handler else self.status_handler.is_reboot_pending
+        reboot_pending = self.package_manager.force_reboot or reboot_pending
```

Comment:
Log again here if the reason why reboot is happening is to help mitigate an issue with the package manager.

Reply:
Done.